### DISCUSSION TITLE
Auswahllisten in mobiler Darstellung

### DISCUSSION THREAD

**Discussion by k-placer** [e-netze/webgis-community#94] [General]
1. Ich tippe bei einem Eingabefeld einen Suchbegriff (z.B. 123) ein

2. Als Ergebnis erhalte ich eine Liste von Werten, die länger als mein Smartphone Display ist und von der Tastatur verdeckt wird
 
![Screenshot_smartphone_keyboard](https://github.com/gis-eni/webgis-forum/assets/97947925/ffdf54ad-3115-47a3-96b3-50362b3a7269)

3. Ich verkleinere die Tastatur. Ich sehe noch immer eine verkürzte Auswahlliste. Die drei Punkte am Ende der Liste lassen vermuten, dass ich hier weitere Werte nachladen kann:
![Screenshot_smartphone_no_keyboard](https://github.com/gis-eni/webgis-forum/assets/97947925/7ad8838a-50ae-4308-ace4-233bae05e932)

4.  Wenn ich auf die 3 Punkte tippe, werden diese 3 Punkte ins Suchfeld übertragen.
<img width="244" alt="Screenshot_smartphone_3punkte" src="https://github.com/gis-eni/webgis-forum/assets/97947925/66806c35-a5ee-441d-907d-2893e8a50eb2">

5. "Suchen"-Button anklicken liefert damit kein Ergebnis, weil der Suchbegriff "..." nicht in den Daten enthalten ist.

Warum kann ich die Vorschlagsliste nicht mit dem Finger nach unten bzw. nach oben verschieben? Wozu dienen die 3 Punkte?

**Comment by gis-eni** [comment]
Die ``...`` scheinen noch ein Relikt aus alten WebGIS Versionen zu sein. Werde schauen, dass wir das los werden, bzw dass die nicht übertragen werden.
Eventuell könnte man das so einrichten, dass beim Klick auf ``...`` ein Popup mit allen Werten (die zur bisherigen Eingabe passen) aufgehen, in dem der User durch die Werte scrollen kann. 
Außerdem sollte man die Liste generell auf 6-7 Werte reduzieren.

**Comment by gis-eni** [comment]
Dauert leider, weil im Zuge der Umsetzung eine neues Autocomplete Control entwickelt werden muss!

**Reply by k-placer** [reply]
Wie ist der Status beim neuen Autocomplete Control?

**Reply by gis-eni** [reply]
Hab ich nicht vergessen. Das "Autocomplete Control" wäre sogar schon fertig. Es ist allerdings nicht ganz trivial, das auch mit dem bestehenden "Typahead Control" auszutauschen. Das ganz ist auf relativ vielen Seiten über die API eingebunden, nicht nur im Viewer. 
Darum muss da noch viel getestet werden :(
Wie es ausschaut kommt das erste mit der WebGIS 6.x Version. Also sobald wir dann endlich Open Source sind...

**Comment by gis-eni** [comment]
Ab Build 5.23.4501:

Es werden alle (bzw. max 250) Ergebnisse angezeigt. Der Ergebnis ist bei vielen Ergebnissen eine scrollbare Liste.
